xbmc/listproviders/DirectoryProvider.cpp

```diff
@@ -37,6 +37,7 @@
 #include "video/VideoInfoTag.h"
 #include "video/VideoThumbLoader.h"
 #include "video/VideoUtils.h"
+#include "video/dialogs/GUIDialogVideoVersion.h"
```

Comment:
@ksooo since you've been working with this stuff. I guess this dialog include should be moved to the some action processor isn't it?

Reply:
As I said, we can fix that after merge of this PR. I currently have not enough time to assist here any further.

Reply:
should a todo comment be added so that it's not forgotten about?